docs/user.md

```diff
@@ -242,3 +242,34 @@ metadata:
 
 Note that timezone required for `timestamp` (offset relative to UTC, see RFC
 3339 section 5.6)
+
+## Increase volume size
+
+PostgreSQL operator supports statefulset volume resize if you're using the
+operator on top of AWS. For that you need to apply manifest with a new size,
```

Comment:
I'd change it something like (not clear about which size you are talking about, could be more than one volume that needs to be changed):

'you need to change the size field of the volume description in the cluster manifest and apply it; the operator compares the new volume size with the previous on and acts on differences.'